lib/customers.js

```diff
@@ -332,4 +338,40 @@ function batch () {
     }, customers)))
 }
 
-module.exports = {add, get, batch, getById, update}
+function updatePhotoCard (id, data) {
+  let imageData = _.get('idCardPhotoData', data)
+
+  if (_.isEmpty(imageData)) {
+    return Promise.resolve(data)
+  }
```

Comment:
It's best to start returning a promise at the beginning of the function and have all the returns inside that block instead of using Promise.resolve.

Reply:
done